### ISSUE TITLE
FR: 'Use Tailscale Subnets' Options for mobile client

### ISSUE THREAD

**Issue by rlaisqls** [tailscale/tailscale#14333]
### What are you trying to do?

I'm configuring routing for a development server using an app connector, and when using an exit node instead of routing directly to the development server, I disable routing via the 'Use Tailscale Subnets' option on the desktop.

I'd like to do the same on mobile, but this option isn't available on either the Android or iOS clients. I’ve come across similar issues, but it looks like this feature still hasn't been implemented:

https://github.com/tailscale/tailscale/issues/2154
https://github.com/tailscale/tailscale/issues/2155

### How should we solve this?

But I want to be able to choose this option on the client.
(I'm not sure if specifying the [MDM option of that](https://tailscale.com/kb/1315/mdm-keys#set-whether-the-device-accepts-tailscale-subnets) as user-decides would allow selection..)

### What is the impact of not solving this?

Mobile clients are not sufficiently versatile for various use cases.

**Comment by agottardo** [comment]
> I'm not sure if specifying the [MDM option of that](https://tailscale.com/kb/1315/mdm-keys#set-whether-the-device-accepts-tailscale-subnets) as user-decides would allow selection

No, it won't, as the option is not implemented in the mobile GUIs. If you set it via an MDM policy to `never`, however, the Tailscale backend will obey it, even on mobile devices. We should add the GUI option.

**Comment by agottardo** [comment]
cc @sonovawolf, not sure what is the right place to add this in our settings
